kubernetes/resource_kubernetes_cron_job_v1_test.go

```diff
@@ -182,6 +185,7 @@ func testAccKubernetesCronJobV1Config_basic(name, imageName string) string {
     concurrency_policy            = "Replace"
     failed_jobs_history_limit     = 5
     schedule                      = "1 0 * * *"
+	timezone 					  = "Etc/UTC"
```

Comment:
CI is complaining about the tab here and in the other places where you've made this change